src/mlpack/methods/ann/convolution_rules/naive_convolution.hpp

```diff
@@ -55,17 +57,21 @@ class NaiveConvolution
               const size_t dW = 1,
               const size_t dH = 1,
               const size_t dilationW = 1,
-              const size_t dilationH = 1)
+              const size_t dilationH = 1,
+              const bool appending = false)
```

Comment:
Do you think we can just remove the `appending` parameter and always assume that it is true?  We cpuld updatw the documentation to note that the convolution is *added* to the output matrix always.  I think nearly everywhere we are using `appending = true` now anyway, and if not, we can just call `.zeros()` first.

Reply:
Oops, I just realized there is an open thread for this already.  Sorry about that...